scanner/config/config.go

```diff
@@ -185,9 +187,22 @@ func (c *MatcherConfig) validate() error {
 			return fmt.Errorf("indexer_addr: failed to parse address: %w", err)
 		}
 	}
+
+	if c.VulnerabilitiesURL == "" {
+		return errors.New("vulnerabilities_url: cannot be empty")
+	}
 	if _, err := url.Parse(c.VulnerabilitiesURL); err != nil {
-		return fmt.Errorf("vulnerabilities_url: %w", err)
+		return fmt.Errorf("vulnerabilities_url: invalid URL: %w", err)
+	}
+
+	v := "dev"
+	if buildinfo.ReleaseBuild {
+		v = version.Version
+	}
+	if c.VulnerabilityVersion != "" {
+		v = c.VulnerabilityVersion
 	}
+	c.VulnerabilitiesURL = strings.ReplaceAll(c.VulnerabilitiesURL, "{{.Version}}", v)
```

Comment:
May be preferred to render this as a template instead of string replace, such as in: https://go.dev/play/p/MCgZ0Okqx4K